index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="If you need a software developer, I'm here for you, just see my work and contact me">
+        <title>Lucas' Portfolio</title>
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <body>
+      <div class="container">
+        <header class="header">
+          <div class="logo"><a href="#">Lucas' Portfolio</a></div>
+          <nav class="icon">
+            <a href="#"><img src="src/Menu.svg" width="24" height="24" alt="Menu icon"></a>
+          </nav>
+        </header>
+<section class="headline">
+<div class="headline-container">
+  <div class="headline-section1">
+    <h1>Hey There.<br/>
+    I'm Lucas</h1>
+    <h2>I'm a Software Developer</h2>
+    <p>I can help you build a product, feature or website Look through
+      some of my work and experience! If you like what you see and have
+       a project you need coded, don't hestiate to contact me.
+    </p>
+  </div>
+  <div class="headline-section2">
+    <ul>
+      <li>
+        <a href="#"><img src="src/github.svg" width="24" height="24" alt="Giuthub logo"></a>
+      </li>
+      <li>
+        <a href="#"><img src="src/Linkedin.svg" width="24" height="24" alt="Linkedin logo"></a>
+      </li>
+      <li>
+        <a href="#"><img src="src/Angelist.svg" width="24" height="24" alt="Angelists logo"></a>
+      </li>
+      <li>
+        <a href="#"><img src="src/twitter.svg" width="24" height="24" alt="Twitter logo"></a>
+      </li>
+      <li>
+        <a href="#"><img src="src/medium.svg" width="24" height="24" alt="Medium logo"></a>
+      </li>
```

Comment:
- [ ] Some icons are not showing on the page. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/148287017-dc5fcbd5-3c22-4158-bb6f-ba2ad285d8f6.png)